api/middleware/logger.go

```diff
@@ -67,6 +65,9 @@ func Logger() macaron.Handler {
 		if len(ctx.Req.Form) > 0 {
 			paramsAsString += "?"
 			paramsAsString += ctx.Req.Form.Encode()
+		} else {
```

Comment:
This empty block is left like this on purpose? do we want to log anything here?

Reply:
Yes, this is empty on purpose. It's documentation for a corner case that isn't handled.